chart/apisix/Chart.yaml

```diff
@@ -31,9 +31,16 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 2.1.0
```

Comment:
chart version is different with application version, our chart is not validated by many production, it is not stable now, so `0.x.x` would be better. `2.1.1` tells people it already production ready.

Reply:
I am going to use the third minor version number as the helm version, keeping the first two consistent with APISIX. 
e.g. 2.1.x for chart version.

Reply:
Ok, use 0.x.x first